libs/newsletter-workflow/src/lib/steps/newsletterData/completeDataCollectionLayout.ts

```diff
@@ -36,7 +38,7 @@ export const completeDataCollectionLayout: WizardStepLayout<DraftStorage> = {
 		back: {
 			buttonType: 'RED',
 			label: 'Back',
-			stepToMoveTo: 'ophan',
+			stepToMoveTo: 'signUpEmbed',
 			executeStep: executeModify,
 		},
 		next: {
```

Comment:
I don't think we need this step anymore, it's not in the index.ts for this folder. I think we can just delete the file.
If we do need it, the next step is still set to 'emailCentralProduction'

Reply:
Updated - cheers